### DISCUSSION TITLE
Modify Spacing Between Equation and Surrounding Text?

### DISCUSSION THREAD

**Discussion by davidleejy** [typst/typst#3406] [Q&A] [answered]
Hi, is there a way to modify the spacing in between an equation and the surround text (space is demarcated with pink shading in the screenshot below)?

Thank you for reading, and have a great day.

 
<img width="614" alt="Screenshot 2024-02-13 at 2 11 37 PM" src="https://github.com/typst/typst/assets/1690072/98324335-9415-4650-992e-594beb591920">

**Comment by xkevio** [comment, marked as answer]
Yes, you can use a show rule on `math.equation` to wrap it in a block and set either the `spacing` parameter for both the distance above and below the equation or set it separately with `above` and `below`.

`#show math.equation: set block(above: 2em, below: 2em)`

**Reply by davidleejy** [reply]
Thank you. Your suggestion worked nicely. 🎆
